include/madrona/mw_gpu.hpp

```diff
@@ -93,8 +116,9 @@ class MWCudaExecutor {
     static CUcontext initCUDA(int gpu_id);
 
     MWCudaExecutor(const StateConfig &state_cfg,
-                                        const CompileConfig &compile_cfg,
-                                        CUcontext cu_ctx);
+                   const CompileConfig &compile_cfg,
+                   const RenderConfig &render_cfg,
```

Comment:
Let's move RenderConfig to the last argument and have it be a default argument that disables the renderer. Trying to minimize churn for existing users.